test/functional/fixtures/regression/gh-1054/test.js

```diff
@@ -0,0 +1,11 @@
+describe('[Regression](GH-1054)', function () {
+    describe('Target element should have the first symbol of typing text when the first input event raised', function () {
```

Comment:
Target element should contain the first symbol of text input when the input event raised at the first time.